admin/tabs/activity/tracking/charts/ChartsModel.js

```diff
@@ -55,24 +59,28 @@ export class ChartsModel extends HoistModel {
                         click: (e) => this.selectRow(e)
```

Comment:
Overall -- This looks good now -- there are a few additional changes to the chart config that are probably unrelated.  

I realize that I mentioned thinking that the 'step' argument is a bit funny, but perhaps we want to defer that change to a seperate PR.

What is the change to `alignTicks` and `startOnTick` is that needed?